features/volgindicaties.feature

```diff
@@ -0,0 +1,154 @@
+# language: nl
+
+Functionaliteit: Als gebruiker wil ik wijzigingen kunnen opvragen van zowel binnengemeentelijke als buitengemeentelijke personen
+    Om wijzigingen te ontvangen van buitengemeentelijke personen, moet een volgindicatie worden gezet in GBA-V middels een Ap01 bericht.
+    Een volgindicatie in GBA-V wordt beëindigd met een Av01 bericht.
+    
+    Hierbij gelden 3 hoofdregels:
+    1. Wanneer er ten minste één gebruiker een actieve volgindicatie heeft op een buitengemeentelijke persoon, moet er ook een actieve volgindicatie zijn voor deze persoon bij het GBA-V.
+    2. Wanneer geen enkele gebruiker een actieve volgindicatie op een buitengemeentelijke persoon, mag er geen actieve volgindicatie zijn voor deze persoon bij het GBA-V.
+    3. Voor een binnengemeentelijke persoon mag er geen actieve volgindicatie zijn in het GBA-V.
+
+   Volgindicaties in GBA-V kunnen worden gewijzigd n.a.v.:
+      - Een PUT /volgindicaties request
+      - Het bereiken van een datum
+      - Een binnengemeentelijke wijziging (de update applicatie ontvangt van een StUF npsLk01 kennisgeving uit het burgerzakensysteem)
+        - persoon is nu ingeschreven in deze gemeenten en applicatie ontvangt StUF mutatiesoort='V': uitverhuizing
+        - persoon is nu ingeschreven in andere gemeenten en applicatie ontvangt StUF mutatiesoort≠'V' (normaliter 'T'): inverhuizing
+ 
+
+   Scenario: Een gebruiker zet als eerste een volgindicatie op een binnengemeentelijk persoon
```

Comment:
Volgens mij is het niet relevant of een gebruiker als eerste een volgindicatie zet op een binnengemeentelijk persoon. Voor een binnengemeentelijk persoon wordt toch nooit een Ap01 bericht gestuurd?

Reply:
Als er al een actieve volgindicatie is bij een andere gebruiker dan wordt er so-wie-so geen Ap01 gestuurd, ook niet voor een buitengemeentelijk persoon. 
In de titel van het scenario is het inderdaad niet zo relevant. Bij Gegeven laat ik dit wel staan